EpsilonCCS/Src/CanParsers/CanParser.c

```diff
@@ -8,43 +8,43 @@
 #include "MpptCanParser.h"
 
 // 0x720 >= AUX_BMS >= 0x723
-#define AUX_BMS_CAN_ID (0x720)
-#define AUX_BMS_CAN_MASK (0xFFC)
+#define AUX_BMS_CAN_MIN (0x720)
+#define AUX_BMS_CAN_MAX (0x72F)
```

Comment:
Should the max not be 0x723 as that is the highest CAN message that would be handled for AUX BMS (same idea for the rest)? Or are you going for the full range of CAN addresses that are assigned to AUX BMS? Either way is fine but it is just strange that the comment with the max is different then the maxes that are defined.